hono/test/utils/factory/items.ts

```diff
@@ -1,20 +1,10 @@
 import { faker } from '@faker-js/faker'
-import { items, itemTypeEnum } from '../../../src/schema'
+import { items, itemTypeEnum } from '../../../src/schema/schema'
+import type { ItemInterface } from '../../../src/schema/zod-definitions'
 import { instance } from '../db'
 
 export type ItemType = 'layer' | 'top' | 'bottom' | 'footwear' | 'accessory'
 
-export interface Item {
```

Comment:
Let me know what you think of me replacing this type definition with a zod type definition imported from zod-definitions. @rak3rman

Reply:
I think this is wise as long as the types are still extended over from the zod defs.